app/controllers/users_controller.rb

```diff
@@ -19,4 +27,19 @@ def change_password
       render json: { errors: { old_password: ["is incorrect"] } }, status: 400
     end
   end
+
+  private
+    def user
+      @user ||= User.find(params[:id])
+    end
+
+    def initialize_user_update_params
+      if @user_update_params
```

Comment:
@jonlemmon, can you explain why `@user_update_params` is a class variable that's set only once? wouldn't this be set based on the `initialized` / `not initialized` of the first updated user and then stay that way?